scripts/cybershake/auto_submit.py

```diff
@@ -56,6 +55,8 @@ def submit_task(
     if not os.path.isdir(ch_log_dir):
         os.mkdir(ch_log_dir)
 
+    params = utils.load_sim_params(os.path.join(sim_dir, "sim_params.yaml"))
```

Comment:
although its unlikely to change, but wouldn't it be better if we use the defined structure from qcore.simulation_structure?

Reply:
Good point, changed